src/mon/AuthMonitor.cc

```diff
@@ -315,8 +314,7 @@ bool AuthMonitor::prepare_update(MonOpRequestRef op)
   case CEPH_MSG_AUTH:
     return prep_auth(op, true);
   default:
-    assert(0);
-    return false;
```

Comment:
see my other comment @ https://github.com/ceph/ceph/pull/9202/commits/8a0d6f1a47b36cc5d2ff0818576ce687ecb0ab0e#r63887619

same applies to the chunk above.